### PR TITLE
Fix avatar in facility list

### LINKED ISSUE #9577: Display Facility Cover Image in Mobile View on the Facility Tab
**Describe the bug**

When we open the facility homepage on a mobile device, the cover image is not visible.

**To Reproduce**

1. Go to the facility homepage.
2. See error

**Expected behavior**

The cover image should be visible in mobile view.

**Screenshots**

Image

### PR DESCRIPTION
**Pull request by AdityaJ2305**
## Proposed Changes

- Fixes #9577 
- Change 1
- Change 2
- More?

@ohcnetwork/care-fe-code-reviewers

## Merge Checklist
- [x] Request for Peer Reviews

## Summary by CodeRabbit

- **New Features**
	- Updated the visibility and layout of links in the FacilityCard component for improved navigation.
- **Bug Fixes**
	- Simplified visibility logic for better responsiveness.

### CODE REVIEW

File: src/components/Facility/FacilityCard.tsx [hunk 1]
```diff
@@ -63,7 +63,7 @@ export const FacilityCard = (props: {
           <div className="h-full w-full grow">
             <Link
               href={`/facility/${facility.id}`}
-              className="group relative z-0 flex w-full min-w-[15%] items-center justify-center self-stretch min-[425px]:hidden"
+              className="group relative z-0 flex w-full min-w-[15%] items-center justify-center self-stretch min-[450px]:hidden"
```
Comment (rithviknishad) [inline src/components/Facility/FacilityCard.tsx]: use rem values over px

https://stackoverflow.com/a/30392895/7887936

**Review by rithviknishad — CHANGES_REQUESTED** [review]

**Comment by bodhish** [comment]
@AdityaJ2305 make sure you complete Merge Checklist

File: src/components/Facility/FacilityCard.tsx [hunk 2]
```diff
@@ -172,6 +161,7 @@ export const FacilityCard = (props: {
                           ),
                       )}
                     </div>
+                    <div className="bg-red-500 w-12">here</div>
```
Comment (rithviknishad) [inline src/components/Facility/FacilityCard.tsx]: ?
```suggestion
```

File: tailwind.config.js [hunk 3]
```diff
@@ -23,6 +23,9 @@ module.exports = {
       fontFamily: {
         sans: ["Figtree", "sans-serif"],
       },
+      screens: {
+        "facility-view": "37rem",
+      },
```
Comment (rithviknishad) [inline tailwind.config.js]: let's not do this. "facility-view" is not a "screen". you can use arbitrary values in the inline classes directly instead

**Review by rithviknishad — CHANGES_REQUESTED** [review]

**Comment by rithviknishad** [comment]
we don't have this page in new arch.

**Pull request closed without merging**